src/repository.rs

```diff
@@ -251,8 +252,59 @@ impl<M: Model> Repository<M> {
     /// ```
     pub async fn bulk_update<V, U>(&self, updates: V) -> Result<BulkUpdateResult>
     where
-        V: Borrow<Vec<U>>,
-        U: Borrow<BulkUpdate>,
+        M: Send + Sync,
+        V: Borrow<Vec<U>> + Send + Sync,
+        U: Borrow<BulkUpdate> + Send + Sync,
+    {
+        Ok(self.coll.bulk_update(&self.db, updates).await?)
+    }
+}
+
+
+#[async_trait]
+pub trait MongodmCollectionExt {
+    async fn bulk_update<V, U>(&self, db: &MongoDatabase, updates: V) -> Result<BulkUpdateResult>
+    where
+        V: 'async_trait + Send + Sync + Borrow<Vec<U>>,
+        U: 'async_trait + Send + Sync + Borrow<BulkUpdate>;
+}
+
+use crate::prelude::MongoDatabase;
+
+#[async_trait]
+impl<M: Send + Sync> MongodmCollectionExt for mongodb::Collection<M> {
+    /// Apply multiple update operations in bulk.
+    ///
+    /// This will be removed once support for bulk update is added to the official driver.
+    /// [see](https://jira.mongodb.org/browse/RUST-531) for tracking progress on this feature in the official driver.
+    ///
+    /// # Example
+    ///
+    /// ```ignore
```

Comment:
I would prefer the example to be at least compiled (if not run).
```suggestion
    /// ```no_run
```

Reply:
That would require having a valid collection object somehow, any idea how?

Reply:
Nvm, fixed.